input/encoder.go

```diff
@@ -59,7 +58,7 @@ func NewEncoder() *FLBEncoder {
 	enc := new(FLBEncoder)
 	enc.handle = new(codec.MsgpackHandle)
 	enc.handle.WriteExt = true
-	enc.handle.SetBytesExt(reflect.TypeOf(FLBTime{}), 0, &FLBTime{})
+	_ = enc.handle.SetBytesExt(reflect.TypeOf(FLBTime{}), 0, &FLBTime{})
```

Comment:
don't shadow errors

Reply:
There was no error handling, and I don't want to break change this package yet. This was just to pass linting. I'll put a code comment I think.